### PR TITLE
ci: line overwrite bypass in res-update

### PR DESCRIPTION
**Pull request by Constrat**
ref:
- https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/e5be6a64f76d811d7641d0c71333a83eb5c04bf6#commitcomment-133674922
- https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/e5be6a64f76d811d7641d0c71333a83eb5c04bf6#commitcomment-133704379
- https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/e5be6a64f76d811d7641d0c71333a83eb5c04bf6#commitcomment-133707192

### CODE REVIEW

File: .github/workflows/res-update-game.yml [hunk 1]
```diff
@@ -88,13 +88,18 @@ jobs:
...
-              $firstNumber = $numbers[$i]
-              $secondNumber = $numbers[$i+1]
+            $firstNumber = $numbers[$i]
+            $secondNumber = $numbers[$i+1]
 
-              if ($firstNumber -ne $secondNumber) {
-                  $isAllEqual = $false
-                  break
-              }
+            if ($firstNumber -gt 1) {
```
Comment (AnnAngela) [inline .github/workflows/res-update-game.yml:94]: `-gt` means `$firstNumber` should be at least `2`, am I right?

Comment (Constrat) [reply .github/workflows/res-update-game.yml:94]: `-gt` -> `Greater than` -> `$firstNumber > 1`

Comment (AnnAngela) [reply .github/workflows/res-update-game.yml:94]: LGTM

**Review by AnnAngela — APPROVED** [review]

**Comment by Constrat** [comment]
Crap wrong branch  😢

**Pull request merged**